mix_tasks/scraper.ex

```diff
@@ -2,13 +2,18 @@ defmodule Location.Scraper do
   @base_url "https://en.wikipedia.org"
   @subdivision_base_url @base_url <> "/wiki/ISO_3166-2:"
   @translations_dest Application.app_dir(:location, "/priv/iso_3166-2.en-translations.json")
+  @version_file Application.app_dir(:location, "/priv/version")
   @countries_to_skip [
     # For Estonia the local names are better than English ones
     "EE",
     # Source data from salsa-debian already has english translations where applicable
     "JP"
   ]
 
+  def write_date_to_version() do
+    File.write!(@version_file, Date.to_iso8601(Date.utc_today()))
```

Comment:
Please make sure that this directory is writable in docker environment. Otherwise perhaps make it configurable so we can set it to `PERSISTENT_CACHE_DIR` to keep it around across restarts, if you think that's a good idea.

Reply:
AFAIK (and I might be wrong) we vendor the list of locations/etc in the library itself and it's never updated in the application - rather when a new version is pulled via `mix deps.get`. 

As such we don't need to update docker environment here.

Reply:
Yup OK, I got confused, sorry for the noise

Reply:
Scraper module is being removed in #20.

Reply:
Does the version need to be a date? Can it be a GIT SHA?

Reply:
Great idea @ruslandoga